RHEL/6/input/checks/avahi_ip_only.xml

```diff
@@ -0,0 +1,43 @@
+<def-group>
+  <definition class="compliance" id="avahi_ip_only" version="1">
+    <metadata>
+      <title>Serve Avahi Only via Required Protocol</title>
+      <affected family="unix">
+        <platform>Red Hat Enterprise Linux 6</platform>
+      </affected>
+      <description>
+      Require Avahi to run on IPv4 or IPv6.
+      </description>
+      <reference source="galford" ref_id="20150316" ref_url="test_attestation" />
+    </metadata>
+    <criteria comment="Check Avahi IPv4 and IPv6 settings" operator="OR">
+      <extend_definition comment="avahi package is not installed" definition_ref="package_avahi_installed" negate="true" />
+      <criteria comment="Test that Avahi IPv4 is enabled" operator="AND">
+        <criterion comment="Avahi IPv4 setting is enabled" test_ref="test_avahi_ipv4_setting" />
+        <criterion comment="Avahi IPv6 setting is disabled" test_ref="test_avahi_ipv6_setting" negate="true" />
+      </criteria>
+      <criteria comment="Test that Avahi IPv6 is enabled" operator="AND">
+        <criterion comment="Avahi IPv4 setting is disabled" test_ref="test_avahi_ipv4_setting" negate="true"/>
+        <criterion comment="Avahi IPv6 setting is enabled" test_ref="test_avahi_ipv6_setting" />
```

Comment:
Sorry for adding 2-nd round (took me a bit to parse this one). Anyway, shouldn't there yet one couple of tests to be added for the case when both IPv4 and IPv6 protocols are enabled on the system? (IOW another test to be added as the last one in the top-level OR checking both ```use-ipv4=yes``` and ```use-ipv6=yes``` options.

The current implementation will return FALSE in this case:
* both IPv4 & IPv6 enabled on the system, and
* both ```use-ipv4=yes``` and ```use-ipv6=yes``` options present in the ```avahi-daemon.conf``` config file.